var/spack/repos/builtin/packages/parmetis/package.py

```diff
@@ -93,11 +92,10 @@ def install(self, spec, prefix):
         if '+gdb' in spec:
             options.append('-DGDB:BOOL=ON')
 
-        with working_dir(build_directory, create=True):
-            cmake(source_directory, *options)
-            make()
-            make('install')
+        return options
 
-            # The shared library is not installed correctly on Darwin; fix this
-            if (sys.platform == 'darwin') and ('+shared' in spec):
-                fix_darwin_install_name(prefix.lib)
+    @run_after('install')
+    def darwin_fix(self):
+        # The shared library is not installed correctly on Darwin; fix this
+        if (sys.platform == 'darwin') and ('+shared' in spec):
```

Comment:
spec is not defined here

Reply:
Thanks for catching this defect.  Some of the Travis checks failed as well.  I'll work on fixing those and push an update.